website/content/labs/lab5.md

```diff
@@ -86,7 +86,7 @@ Before continuing, take a moment to look at the `Makefile` contained inside the
 
```

Comment:
I'm not getting that warning when running on a CS machine, but I've nonetheless replaced `abs` with `fabs` in the starter code. Can you check whether that clears out the warning on your end?

Reply:
Whoops, should've run on a CS machine as well. Reran on a CS machine instead of my local Ubuntu 19.10 environment and didn't get the warning when building.

`fabs()` also does the trick for removing the warning for my local environment (and the CS machine as well)